### DISCUSSION TITLE
repo name change after setting remote origin with another repo prefix

### DISCUSSION THREAD

**Discussion by alirezaaf12** [community/community#42353] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

I struggled too much should I post this or not but it was unique and i had no idea what is going on

so I created my new private repo, then in git cmd , added remote origin like this :

`git remote add origin git@github.com:alirezaaf12/shirintech.git`

but when i use `git remote -v`

this shows up

```
origin  git@github.com:alirezaaf12/emerald.gitalirezaaf12/shirintech.git (fetch)
origin  git@github.com:alirezaaf12/emerald.gitalirezaaf12/shirintech.git (push)
```

i have another private repo named alirezaaf12/emerald.git but i dont understand why it becomes prefix to repo origin

i changed repo origin a lot of times but every time its the same result 

anyone faced something like this? or there is an option to turn this prefix off?

**Comment by airtower-luna** [comment]
That really shouldn't happen, my best guess is that there's some kind of copy & paste error with the URL. How are you copying it?

**Reply by alirezaaf12** [reply]
may be its a problem with my ssh on windows, but didn't find any solution on my system, the work around this problem  found  later was use https git link but surprisingly after that instead of showing 

```
origin https://github.com/alirezaaf12/shirintech.git (fetch) 
origin https://github.com/alirezaaf12/shirintech.git (push) 
```

showed

```
origin git@github.com:alirezaaf12/shirintech.git (fetch) 
origin git@github.com:alirezaaf12/shirintech.git (push) 
```

instead, which is a new problem, i can push to repo but not through https, and i don't know what to do with this ssh problem, kind of new ,may be with windows git , because never encountered with this type of problem on Ubuntu machine

**Reply by airtower-luna** [reply]
Could there be *anything* that works on the Git repository (and possibly changes its settings) other than the Git CLI? :thinking:

**Comment by alirezaaf12** [comment]
examined all options on GitHub repo settings, but didn't found anything that helps, maybe its a particular option, and I don't know what is it
I will research for this problem to find a solution, may be it helps others later

**Comment by alirezaaf12** [comment, marked as answer]
deleted git config file in 

`C:\Users\loginname\.gitconfig 
`
and everything worked as like before
